src/components/MainNav.jsx

```diff
@@ -65,26 +74,34 @@ const MainNav = ({ page, query }) => {
             ></path>
           </svg>
         </div>
-        <div className="dropdown-menu z-10 top-full right-1/2 transform translate-x-1/2 absolute hidden h-auto pt-4 sm:right-auto sm:left-0 sm:translate-x-0 lg:right-0 lg:left-auto">
+        <div className="dropdown-menu z-10 top-full right-1/2 transform translate-x-1/2 absolute hidden h-auto pt-4 sm:right-0 sm:translate-x-0 lg:left-auto">
           <div className="block w-full bg-white shadow-dropdown px-4 py-4 rounded-md text-align-left">
-            <div className="pb-6">
+            <div className="pb-6 w-60 sm:w-96">
               <span className="block mb-2 font-bold text-black text-2xl">
-                Mis Descargas
+                Mi Historial
               </span>
-              {userProductsHistory ? (
-                <div className="pb-4 text-black">
+              {userArtworkHistory ? (
+                userArtworkHistory.map((artwork) => (
+                  <Link 
+                    to={`/artwork/${artwork.id}`} 
+                    className="flex items-center justify-between py-1 hover:underline" key={`historial-item-${artwork.id}`}
+                  >
+                    <p className="text-xl truncate mr-2">{artwork.title}</p>
+                    <img
+                      alt="Arte"
+                      src={artwork?.images?.web?.url}
+                      className="object-cover w-20 h-20"
+                    />
+                  </Link>
+                ))
```

Comment:
Can you please move this to another component? Some like: HistoryItem

Reply:
done